docs/application/native/guides/device/system.md

```diff
@@ -572,6 +572,7 @@ The following table lists the security feature keys.
 | Key                                      | Type   | Description                              |
 |------------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/security.device_certificate` | `bool` | The platform returns `true` for this key, if the device has the Device Certificate its corresponding private key which are used for identifying the device. |
+| `http://tizen.org/feature/security.privacy_privilege` | `bool` | The platform returns true for this key, if the device supports the privacy privilege information feature to ask user to confirm the privilege status. If it is true, the user can turn on or off each application's own user-settable privacy privileges. |
```

Comment:
The platform returns true for this key, => The platform returns `true` for this key,

Reply:
Please rephrase to:

The platform returns `true` for this key, if the device supports the privacy privilege information feature to ask user to confirm the privilege status. If it is `true`, the user can turn on or off the user-settable privacy privileges for each application.

Reply:
Global Comment: Under Feature Keys, please add a colon (:) instead of a period at the end of the sentence. For example, 
The following table lists the network feature keys.
Modify to:
The following table lists the network feature keys: